### ISSUE TITLE
Debian package does not start in master branch

### ISSUE THREAD

**Issue by spinscale** [elastic/elasticsearch#17084]
I just tried to build the debian package and start it on ubuntu 12 LTS. This is the error I got, when running the start-stop-daemon command manually, otherwise the error is concealed.

```bash
start-stop-daemon -d /usr/share/elasticsearch --start --user elasticsearch -c elasticsearch --pidfile /var/run/elasticsearch/elasticsearch.pid --exec /usr/share/elasticsearch/bin/elasticsearch -- -d -p /var/run/elasticsearch/elasticsearch.pid --default.path.home=/usr/share/elasticsearch --default.path.logs=/var/log/elasticsearch --default.path.data=/var/lib/elasticsearch --default.path.conf=/etc/elasticsearch
root@vagrant-ubuntu-precise-64:~# Starts elasticsearch

Option           Description
------           -----------
-D               Configures an Elasticsearch setting
-V, --version    Prints elasticsearch version
                   information and exits
-d, --daemonize  Starts Elasticsearch in the background
-h, --help       show help
-p, --pidfile    Creates a pid file in the specified
                   path on start
-s, --silent     show minimal output
-v, --verbose    show verbose output
ERROR: default.path.home is not a recognized option
```

**Comment by spinscale** [comment]
looks similar for the RPM, but did not test it

**Comment by jasontedor** [comment]
This is a consequence of #17024 and I opened #17087 to address. Note that even with the fix for this particular issue, starting Elasticsearch as a service will still fail because of #16579.

**Comment by jasontedor** [comment]
Closed by #17087.
